static/js/components/dashboard/CourseListCard_test.js

```diff
@@ -58,6 +59,7 @@ describe('CourseListCard', () => {
             coursePrice={coursePrice}
             addCourseEnrollment={() => Promise.resolve()}
             prices={prices}
+            ui={{ showEnrollPayLaterSuccess: false }}
```

Comment:
@noisecapella i am getting flow warnings here, it is excepting all from `UIState`;